sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -369,6 +370,53 @@ def has_ontime_pane(self):
     return False
 
 
+class AfterSynchronizedProcessingTime(TriggerFn):
```

Comment:
Do we want users to use this directly, or should we make it private?

Reply:
I'm rusty on what this entails. How to best expose it to `core.py` without putting in `__all__`?

Reply:
Given you're already doing a local import, I would be fine with importing it directly from here rather than putting it in `__all__`. (That and we could add a leading underscore.)

Reply:
Done